src/use_cases_execution/warehouse/operaton/fetch_incidents.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'logger'
+require 'bas/shared_storage/postgres'
+
+require_relative '../config'
+require_relative '../../../implementations/fetch_records_from_operaton'
+
+read_options = {
+  connection: Config::Database::CONNECTION,
+  db_table: 'warehouse_sync',
+  avoid_process: true,
+  where: 'archived=$1 AND tag=$2 ORDER BY inserted_at DESC',
+  params: [false, 'FetchIncidentsFromOperaton']
+}
+
+write_options = {
+  connection: Config::Database::CONNECTION,
+  db_table: 'warehouse_sync',
+  tag: 'FetchIncidentsFromOperaton'
+}
+
+first_day_of_current_month = DateTime.new(
+  Date.today.year,
+  Date.today.month, 1, 0, 0, 0, '-0'
+).strftime('%Y-%m-%dT%H:%M:%S.%L%z')
+
+process_options = {
+  entity: 'operaton_incident',
+  endpoint: 'history/incident',
+  method: :get,
+  params: {
```

Comment:
Here you are using params, but on the other modules (`activities` and `processes`) you are using body, is this intentional? 🤔